client/state/selectors/is-nav-unification-enabled.js

```diff
@@ -24,6 +29,12 @@ export default ( state ) => {
 		return false;
 	}
 
+	// Users belonging to the current segment OR New Users.
+	const userId = getCurrentUserId( state );
+	if ( userId % 100 < CURRENT_ROLLOUT_SEGMENT_PERCENTAGE || userId >= NEW_USER_ID_THRESHOLD ) {
```

Comment:
Nitpick: We could reuse the `isNavUnificationNewUser` selector here so `NEW_USER_ID_THRESHOLD` is maintained in one single place.

Reply:
Very good point, thanks! 92a7a78